CMSgov/mint-app
# Add communicationMethodOther field to Participants and Providers

## Changes and Description

- communicationMethod Other field added to objects and SQL scripts



## How to test this change

`scripts/dev db:seed`
1.  run this update script to verify that the field can be queryed and updated
```gql
mutation updatePlanParticipantsAndProviders #($input: PlanMilestonesInput!) 
{
    updatePlanParticipantsAndProviders (id: "b18d2f8c-e77b-4cbb-b992-25b43bec9d4f",changes:     
    {


    #Page 1
    # participants:
    # medicareProviderType:
    statesEngagement: "Very good engagement"
    # participantsOther:
    # participantsNote:
    participantsCurrentlyInModels: null
    # participantsCurrentlyInModelsNote:
    # modelApplicationLevel:

    #Page 2
    # expectedNumberOfParticipants:
    # estimateConfidence:
    # confidenceNote:
    # recruitmentMethod:
    # recruitmentOther:
    # recruitmentNote:
    # selectionMethod:
    # selectionOther:
    # selectionNote:

    #Page 3
    # communicationMethod:
    communicationMethodOther: "VA notify application"
    # communicationNote:
    # participantAssumeRisk:
    # riskType:
    # riskOther:
    # riskNote:
    # willRiskChange:
    # willRiskChangeNote:

    #Page 4
    # coordinateWork:
    # coordinateWorkNote:
    # gainsharePayments:
    # gainsharePaymentsTrack:
    # gainsharePaymentsNote:
    # participantsIds:
    # participantsIdsOther:
    # participantsIDSNote:

    #Page 5
    # providerAdditionFrequency:
    # providerAdditionFrequencyOther:
    # providerAdditionFrequencyNote:
    # providerAddMethod:
    # providerAddMethodOther:
    # providerAddMethodNote:
    # providerLeaveMethod:
    # providerLeaveMethodOther:
    # providerLeaveMethodNote:
    # providerOverlap:
    # providerOverlapHierarchy:
    # providerOverlapNote:





    }) {
        id
        modelPlanID
        participants
        medicareProviderType
        statesEngagement
        participantsOther
        participantsNote
        participantsCurrentlyInModels
        participantsCurrentlyInModelsNote
        modelApplicationLevel
        expectedNumberOfParticipants
        estimateConfidence
        confidenceNote
        recruitmentMethod
        recruitmentOther
        recruitmentNote
        selectionMethod
        selectionOther
        selectionNote
        communicationMethod
        communicationMethodOther
        communicationNote
        participantAssumeRisk
        riskType
        riskOther
        riskNote
        willRiskChange
        willRiskChangeNote
        coordinateWork
        coordinateWorkNote
        gainsharePayments
        gainsharePaymentsTrack
        gainsharePaymentsNote
        participantsIds
        participantsIdsOther
        participantsIDSNote
        providerAdditionFrequency
        providerAdditionFrequencyOther
        providerAdditionFrequencyNote
        providerAddMethod
        providerAddMethodOther
        providerAddMethodNote
        providerLeaveMethod
        providerLeaveMethodOther
        providerLeaveMethodNote
        providerOverlap
        providerOverlapHierarchy
        providerOverlapNote
        createdBy
        createdDts
        modifiedBy
        modifiedDts
        status
    }
}
```

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
